src/adapter/por.ts

```diff
@@ -7,6 +7,14 @@ import { EmptyInputParameters, InputParametersDefinition } from '../validation/i
 import { AdapterRequest, AdapterResponse } from '../util'
 import { metrics } from '../metrics'
 
+export type PoRProviderResponse = {
+  Result: number
+  Data: {
+    result: number
+    ripcord: boolean
```

Comment:
Is there a `ripcordMessage` that we can rely on always being included that provides details on why the ripcord is true?

Not a blocker, as we probably don't want to enforce that as a required property (but still include when available)

Reply:
In theory there should be `ripcordDetails` but i wasn't able to get a response with `ripcord=true` to check whether details are provided or not. I think once we have multi-variable feeds, we can logger.warn the ripcordDetails. Not sure if it makes sense to actually return that value as response. In any case the types here just require `result` and `ripcord`, additional ones can be added.

Reply:
That sounds good to me 👍